src/goto-programs/builtin_functions.cpp

```diff
@@ -589,7 +589,9 @@ void goto_convertt::do_function_call_symbol(
       throw id2string(base_name) + " expected not to have LHS";
     }
   }
-  else if(base_name == "__VERIFIER_error" || base_name == "reach_error")
+  else if(
+    base_name == "__VERIFIER_error" || base_name == "reach_error" ||
+    base_name == "__builtin_unreachable")
```

Comment:
Please confirm that __builtin_unreachable should be treated as an error, instead of an abort.

Reply:
[GCC](https://gcc.gnu.org/onlinedocs/gcc/Other-Builtins.html) says it is a UB.